tasks/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
-- ansible.builtin.import_tasks: install.yml
-- ansible.builtin.import_tasks: db/db.yml
-- ansible.builtin.import_tasks: update.yml
+- name: Import install
```

Comment:
I've always left import tasks nameless as the code reads exactly what it's doing versus essentially adding comments to it.

If it's required to pass ansible-lint eventually without disabling (in the future when moving into a role) then please update these to single qouted strings to be consistent with the rest of the role.

Reply:
ansible-lint requires task name, whatever obvious task can be or not
name[missing]: https://ansible.readthedocs.io/projects/lint/rules/name/

The quotes are not required in my xp.